app/src/main/java/nl/avans/drivioapp/MyCarDetailsFragment.kt

```diff
@@ -0,0 +1,81 @@
+package nl.avans.drivioapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.setFragmentResult
+import androidx.fragment.app.setFragmentResultListener
+import androidx.fragment.app.viewModels
+import nl.avans.drivioapp.databinding.FragmentMyCarDetailsBinding
+import nl.avans.drivioapp.viewModel.MyCarsViewModel
+
+class MyCarDetailsFragment : Fragment(R.layout.fragment_my_car_details) {
+    private var _binding: FragmentMyCarDetailsBinding? = null;
+    private val binding get() = _binding!!;
+    private val myCarsViewModel: MyCarsViewModel by viewModels()
+    private var carId: Int? = null;
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentMyCarDetailsBinding.inflate(inflater, container, false);
+        val view = binding.root;
+        return view;
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val tvCarId: TextView = binding.tvCarId
+        val tvBrand: TextView = binding.tvBrand
+        val tvBuildYear: TextView = binding.tvBuildYear
+        val tvModel: TextView = binding.tvModel
+        val tvFuelType: TextView = binding.tvFuelType
+
+        setFragmentResultListener("carId") { requestKey, bundle ->
+            carId = bundle.getInt("carId")
+            myCarsViewModel.getElectricCarById(carId!!)
+
+//            binding.btnDeleteCar.setOnClickListener {
+//                myCarsViewModel.deleteElectricCar(carId!!)
+////                TimeUnit.SECONDS.sleep(1)
+////                replaceFragment(MyCarsFragment())
+//            }
```

Comment:
Kan eruit als het niet gebruikt wordt? Of moet dit nog uitgewerkt worden?

Reply:
Dit is de delete welke ik gisteren nog niet helemaal werkend kreeg, vandaar dat ik hem nu even uitgecomment heb. Ik kan hem er ook uithalen en in een nieuwe branch zetten.

Reply:
Check, is prima zo!